src/util/MongoBinaryDownloadUrl.js

```diff
@@ -200,6 +200,10 @@ export default class MongoBinaryDownloadUrl {
       name += '1604';
     } else if (majorVer === 16) {
       name += '1604';
+    } else if (os.release === '18.04') {
```

Comment:
Please remove this block. 
Check by major version will be enough.

Reply:
ok